### PR TITLE
Updated app.py to handle different error messages based on HTTP requests

### PR DESCRIPTION
**Pull request by mfturra**
App IP address was changed to receive inbound HTTP requests. Error messages were configured to handle different scenarios based on the HTTP request.

### CODE REVIEW

**Comment by mfturra** [comment]
Final changes that were made for the apache build

**Comment by fahlmant** [comment]
Can you update the description to explain what is changing and why?

File: app.py [hunk 1]
```diff
@@ -51,6 +51,24 @@ def videogame_POST():
         error_message = {}
         error_message["endpoint"] = "/videogames"
 
+        if isinstance(data, list):
+            # Evaluate each item in the list
+            for item in data:
+                process_videogame(item)
+
+        elif isinstance(data, dict):
+            process_videogame(data)
```
Comment (fahlmant) [inline app.py:60]: What are these changes for?

**Comment by fahlmant** [comment]
Is this only for MacOS? Does this work on linux too?

**Review by fahlmant — APPROVED** [review]

**Pull request merged**